### ISSUE TITLE
Clean-up releases (download links)

### ISSUE THREAD

**Issue by wernight** [buildout/buildout#163]
The releases are a bit in a mess at the moment. Let me explain my problem with them.
- https://github.com/buildout/buildout advertises for version 2.0 but it has also a release 2.1 (why advertise the older release? may be there is a good reason, I just don't know)
- http://www.buildout.org/en/latest/install.html advertises for a version 2.1 from zope which isn't the latest snapshot on GitHub master branch (which is it?)
- http://downloads.buildout.org/2.1/ is a stable 2.1 release; why advertise the older 2.0 release? May be there is a good reason, I just don't know.
- The latest is a beta 2.2 but in the source code it's marked as 2.1, so there are actually plenty 2.1 versions (the 2.1 "release", the 2.2.0b1 "release", the 2.1 snapshots of various github branches, and 2.1 from http://svn.zope.org/zc.buildout/trunk/bootstrap/bootstrap.py?view=co)
- https://github.com/buildout/buildout/blob/master/CHANGES.rst talks about release 2.2, but where is it?

In short the version in the snapshot is not reliable, there are tons of 2.1 out there, 2.0 and 2.1 stable releases exist but some place advertise to use the master on GitHub others the trunk on Zope. So when I have a bootstrap.py I don't even know if it's outdated or not (when it's 2.1 version).

I'd suggest to mark the version under source control as "snapshot" or "X.X-snapshot". Also I'd change the home page and GitHub readme to give two links: One for the latest suggested stable (either the 2.0 or 2.1) and one for the latest recommended "snapshot" (if any).

**Comment by reinout** [comment]
The latest bootstrap.py now has a version. See https://bootstrap.pypa.io/bootstrap-buildout.py

`python bootstrap.py --version` even works :-)

There's still documentation work/cleanup to be done, though, so I'm keeping this issue open.
